ui/src/index.css

```diff
@@ -1,46 +1,6 @@
-@font-face {
```

Comment:
I think this file should not be touched at all in this PR, and if so, font definitions are gone.

Reply:
Those are long gone - `esbuild` detected the problem that we have not noticed before.

Reply:
They are not gone, and they are required.